Codeinwp/neve
# feat: locked layout for the woocommerce checkout layouts

### Summary

New locked layout added for the WooCommerce Checkout Layout Types.

### Will affect visual aspect of the product

NO

### Screenshots 

### Test instructions

- make sure there is no regression once the free->pro upgrade. (wc checkout layouts should worked well after the pro upgrade)


Closes codeinwp/neve-pro-addon#1723.
